src/test/java/org/broadinstitute/consent/http/db/DAOTestHelper.java

```diff
@@ -248,4 +241,40 @@ private DataAccessRequest createDataAccessRequest(Integer userId, Integer collec
     return dataAccessRequestDAO.findByReferenceId(referenceId);
   }
 
+  User createUserWithRoleInDac(Integer roleId, Integer dacId) {
```

Comment:
These methods were duplicated in subclasses; this change puts the shared code in the base class.